### PR TITLE
Update Android default permissions documentation

### PR DESCRIPTION
**Pull request by gregblass**
Add message about the default permissions that are enabled and explain about WAKE_LOCK

### CODE REVIEW

File: versions/v22.0.0/guides/configuration.md [hunk 1]
```diff
@@ -276,35 +276,51 @@ The following is a list of properties that are available for you under the `"exp
...
+      
+        - `CAMERA`
+        - `READ_CONTACTS`
+        - `READ_PHONE_STATE`
+        - `RECORD_AUDIO`
+        - `WAKE_LOCK`
+    
+      We leave these enabled by default to avoid app crashes due to commonly used component requirements. However, you may not want to have your app request these permissions in production if they aren't needed.
+    
+      As of Android 6 (API level 23), WAKE_LOCK is classed as a "normal" permission, meaning the permission is automatically granted. Removing the WAKE_LOCK permission will often cause apps to crash, so we do not recommend removing it.
```
Comment (brentvatne) [inline versions/v22.0.0/guides/configuration.md]: @satya164 - should we update default permissions for this?

Comment (satya164) [reply versions/v22.0.0/guides/configuration.md]: yeah, we should. I'll do that

Comment (satya164) [reply versions/v22.0.0/guides/configuration.md]: Looking at the code, actually `WAKE_LOCK` is already a required permission and doesn't have to be added to the permission list.

Comment (gregblass) [reply versions/v22.0.0/guides/configuration.md]: Hey @satya164 - I submitted my first app to Google play last week and was surprised to have  4 permissions I wasn't using show up. Then, when I set the permissions to [], my app froze. When I debugged it, it was due to the wake lock.

Comment (gregblass) [reply versions/v22.0.0/guides/configuration.md]: To successfully launch without those permissions, and without my app freezing due to the wake lock, the permissions I had to use ended up being: ["WAKE_LOCK"]

Comment (gregblass) [reply versions/v22.0.0/guides/configuration.md]: None of this was documented. I really think it should be. All of it was a total surprise to me and took hours of my time to figure out.

**Comment by jesseruder** [comment]
`WAKE_LOCK` is now a default permission, so standalone apps shouldn't break in this way:
https://github.com/expo/universe/blob/master/exponent/template-files/android/AndroidManifest.xml#L17

Thanks for pointing this out!

**Comment by gregblass** [comment]
You're very welcome @jesseruder @brentvatne @satya164 !!! Thank you guys for all you do!

**Pull request closed without merging**